lib/config/httpRoutes.js

```diff
@@ -209,47 +208,45 @@ module.exports = [
 
 
   // DELETE
-  {verb: 'delete', url: '/:index/:collection/_truncate', controller: 'collection', action: 'truncate'},
   {verb: 'delete', url: '/:index/:collection/_specifications', controller: 'collection', action: 'deleteSpecifications'},
+  {verb: 'delete', url: '/:index/:collection/_truncate', controller: 'collection', action: 'truncate'},
 
-  {verb: 'delete', url: '/:index/:collection/_query', controller: 'document', action: 'deleteByQuery'},
   {verb: 'delete', url: '/:index/:collection/:_id', controller: 'document', action: 'delete'},
+  {verb: 'delete', url: '/:index/:collection/_query', controller: 'document', action: 'deleteByQuery'},
 
-  {verb: 'delete', url: '/_mdelete', controller: 'index', action: 'mDelete'},
   {verb: 'delete', url: '/:index', controller: 'index', action: 'delete'},
+  {verb: 'delete', url: '/_mdelete', controller: 'index', action: 'mDelete'},
 
-  {verb: 'delete', url: '/roles/:_id', controller: 'security', action: 'deleteRole'},
   {verb: 'delete', url: '/profiles/:_id', controller: 'security', action: 'deleteProfile'},
+  {verb: 'delete', url: '/roles/:_id', controller: 'security', action: 'deleteRole'},
   {verb: 'delete', url: '/users/:_id', controller: 'security', action: 'deleteUser'},
 
+  {verb: 'delete', url: '/ms/:_id', controller: 'ms', action: 'del'},
+  {verb: 'delete', url: '/ms', controller: 'ms', action: 'del'},
   {verb: 'delete', url: '/ms/_hdel/:_id', controller: 'ms', action: 'hdel'},
   {verb: 'delete', url: '/ms/_lrem/:_id', controller: 'ms', action: 'lrem'},
   {verb: 'delete', url: '/ms/_srem/:_id', controller: 'ms', action: 'srem'},
   {verb: 'delete', url: '/ms/_zrem/:_id', controller: 'ms', action: 'zrem'},
   {verb: 'delete', url: '/ms/_zremrangebylex/:_id', controller: 'ms', action: 'zremrangebylex'},
   {verb: 'delete', url: '/ms/_zremrangebyscore/:_id', controller: 'ms', action: 'zremrangebyscore'},
-  {verb: 'delete', url: '/ms/:_id', controller: 'ms', action: 'del'},
-  {verb: 'delete', url: '/ms', controller: 'ms', action: 'del'},
 
 
-  // PUT
+  // PUT (idempotent)
   {verb: 'put', url: '/_updateSelf', controller: 'auth', action: 'updateSelf'},
 
   {verb: 'put', url: '/:index/:collection', controller: 'collection', action: 'create'},
```

Comment:
Since you moved `index:create` to `POST` actions, shouldn't `collection:create` be moved there too?